sass/_SimpleFilter.scss

```diff
@@ -166,8 +169,22 @@
   margin: 0 5px;
   @include justify-content(flex-end);
   color: $color-greyish-teal-blue;
-  width: 10px;
-  height: 6px;
+  width: 12px;
+  height: 8px;
+}
+
+.coveo-simplefilter-eraser {
+  position: relative;
+  top: 1px;
+  margin: 0 5px 0 10px;
+  cursor: pointer;
+  pointer-events: auto;
```

Comment:
Why is this needed ?

Reply:
I spent quit a few time figuring out why the clear button wasn't triggering the click event, an it was because the `.coveo-simplefilter-select` has `pointer-events: none;` which gets inherited to the children. So `pointer-events: auto;` in a child element allow the click event to be properly triggered.

Reply:
👍